src/components/Loading/index.js

```diff
@@ -1,9 +1,43 @@
 import React from 'react';
+import CircleProgressBar from 'components/CircleProgressBar';
+import styles from './loading.scss';
 
-const Loading = ({ isLoading, children, onServer }) => (
-  onServer || (__CLIENT__ && isLoading) || !children
-    ? <div>Loading screen</div>
-    : children
-);
+const Loading = React.createClass({
+
+  getInitialState () {
+    return { percentage: 75 };
+  },
+
+  tick () {
+    this.setState({ percentage: this.state.percentage + 125 });
```

Comment:
The three navigation actions and 2 I thought were a bit overkill (taking a look at the actions now, they’re used in multiple places so maybe it actually makes sense to create actions. :))
- Update current city (petitions actions)
- Update suggestion input value